agent/src/com/thoughtworks/go/agent/DefaultBuildStateReporter.java

```diff
@@ -15,7 +15,6 @@
  * ************************GO-LICENSE-END***********************************/
```

Comment:
I meant, its the old style copyright with `GO-LICENSE-END`. We have been moving the files we touch to the new one such as the one you have in AgentWebSocketClientController.java. Also, files like BuildRepositoryRemoteAdapter.java don't seem to have a copyright notice.